src/symbols.cs

```diff
@@ -0,0 +1,215 @@
+using CoreFoundation;
+using ObjCRuntime;
+using Foundation;
+
+using System;
+
+namespace Symbols {
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolEffect : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolContentTransition : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface NSSymbolEffectOptions : NSCopying, NSSecureCoding {
+		[Static]
+		[Export ("options")]
+		NSSymbolEffectOptions Create ();
+
+		[Static]
+		[Export ("optionsWithRepeating")]
+		NSSymbolEffectOptions CreatehRepeating ();
+
+		[Export ("optionsWithRepeating")]
+		NSSymbolEffectOptions GetWithRepeating ();
```

Comment:
All the methods follow the same pattern and need the same change...

```suggestion
		NSSymbolEffectOptions GetRepeating ();
```